apps/03TicTacToe.js

```diff
@@ -40,10 +44,38 @@ function printBoard() {
   console.log('2 ' + board[2].join(' | '));
 }
 
-function ticTacToe(row, column) {
+function getPrompt() {
+  console.log("===============================");
+  console.log("It's Player " + playerTurn + "'s turn.");
+  console.log();
+  printBoard();
+  prompt.get(['row', 'column'], function(error, result) {
+    var row = parseInt(result['row']);
+    var column = parseInt(result['column']);
+    //check for validity of row and column player move
+    if(validVal(row) && validVal(column) ){
+      ticTacToe(row, column);
+      getPrompt();
+    }
+    else{
+      console.log('invalid entry, try again');
+      getPrompt();
```

Comment:
Looks great!!! I am looking at your code on my desktop as well. I will run it later today when I am off my network.